### PR TITLE
Fix CPBlue LFCLKSRC; CPB has no status neopixel

### PR DESCRIPTION
**Pull request by dhalbert**
- Circuit Playground Bluefruit does not have a 32kHz crystal on board. This is quite unusual, and the nRF clock code assumed there would be one. Caused BLE failures and other timing-related issues.
- There is not a status neopixel on this board, as with CPX: first neopixel was getting reset with status colors.
- Fixed typo in an unrelated error string.

Thanks @ladyada and @hathach.

### CODE REVIEW

**Comment by dhalbert** [comment]
@bmeisels and @jepler Apologies: I encountered exactly the same issue with the Circuit Playground Bluefruit and didn't realize there was a PR approved 9 hours beforehand. I wrote essentially the same PR, but with just `#if` and no inline routine. This is more the style we use now. Also fixed a couple of other things at the same time.

File: ports/nrf/common-hal/bleio/Adapter.c [hunk 1]
```diff
@@ -39,37 +39,26 @@
 #include "supervisor/usb.h"
 #include "shared-bindings/bleio/Adapter.h"
 #include "shared-bindings/bleio/Address.h"
-#include "mpconfigboard.h" // for BOARD_HAS_CRYSTAL
```
Comment (dhalbert) [inline ports/nrf/common-hal/bleio/Adapter.c]: Note that `mpconfigboard.h` is already included via the other includes.

File: ports/nrf/mpconfigport.h [hunk 2]
```diff
@@ -38,6 +38,11 @@
 #define MICROPY_PY_UBINASCII                     (1)
 #define MICROPY_PY_UJSON                         (1)
 
+#ifndef BOARD_HAS_32KHZ_XTAL
+// Assume crystal is present, which is the most common case.
+#define BOARD_HAS_32KHZ_XTAL (0)
```
Comment (hathach) [inline ports/nrf/mpconfigport.h]: the comment seem to contradict the value

Comment (dhalbert) [reply ports/nrf/mpconfigport.h]: RIghto :face_with_head_bandage:

**Comment by dhalbert** [comment]
Added some `#pragma message ()` temporarily to check that the correct `#if BOARD_HAS_32KHZ_CRYSTAL` conditional arms were being used.

I should not try to complete a PR after midnight.

**Pull request merged**